compiler/src/dotty/tools/dotc/transform/init/Objects.scala

```diff
@@ -1054,6 +1046,8 @@ class Objects(using Context @constructorOnly):
         else if target.equals(defn.Predef_classOf) then
           // Predef.classOf is a stub method in tasty and is replaced in backend
           UnknownValue
+        else if target.is(Flags.JavaDefined) then
```

Comment:
I added the modification here to skip java-defined methods. It is worth noting that java-defined methods has source, but its defTree is incomplete, so we should not evaluate them anyway

Reply:
I don't understand what way the `defTree` is incomplete. The definition of `hasSource` is:
```scala
    def hasSource(using Context): Boolean = !sym.defTree.isEmpty
```

But anyway, perhaps `hasSource` should check for Java-defined and return false for all Java-defined symbols. @liufengyun what do you think?

Reply:
> It's a good idea to change `hasSource` such that it return false for all Java-defined symbols.

Done